app/Http/Controllers/MapController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * MapController.php
+ *
+ * Controller for graphing Relationships
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2019 Thomas Berberich
+ * @author     Thomas Berberich <sourcehhdoctor@gmail.com>
+ */
+
+namespace App\Http\Controllers;
+
+use App\Models\Device;
+use Illuminate\Http\Request;
+use LibreNMS\Config;
+use LibreNMS\Util\Url;
+
+class MapController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('deny-demo');
+    }
+
+    protected function visOptions()
+    {
+        return Config::get('network_map_vis_options');
+    }
+
+    protected function nodeDisabledStyle()
+    {
+        return array('color' => array(
```

Comment:
you have a mix between `array()`and `[]`, stick to the newer `[]` style

Reply:
done